### DISCUSSION TITLE
Suggestion: Checkpoints for 1.41.0

### DISCUSSION THREAD

**Discussion by Piggymaster1222** [JordanSantiagoYT/FNF-JS-Engine#827] [Ideas]
it basically goes like this: have you ever died during a section of the song that you keep dying on? well with the checkpoint update, you can no longer waste your time restarting and can pick up where you left off!

**Comment by imavie** [comment]
I think you can do it in LUA

**Comment by TheStinkern** [comment]
This would make the score and accuracy be off
